### PR TITLE
add a close button on how to play

### LINKED ISSUE #652: A small close button on top left corner of how to play page should be there to close it
### Expected results
A small close button on top left corner of how to play page should be there to close it in addition to play button at the bottom.
### Actual results
No such close button

### Would you like to work on this
Yes

### PR DESCRIPTION
**Pull request by PradumanPannu**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [ ] Squashed the commits.

### Changes done in this Pull Request

- **Test-Link**
  http://rawgit.com/PradumanPannu/labyrinth/b2/howtoplay.html

- Fixes #652 

### Screenshots,
close button

### CODE REVIEW

File: howtoplay.html [hunk 1]
```diff
@@ -16,7 +16,8 @@
 </head>
 
 <body>
-  <div class="whitespace">
+    <a onclick="javascript: document.location='index.html'" id="Close-Button">&#10006;</a>
```
Comment (abishekvashok) [inline howtoplay.html]: It's inline js, can we seperate it?

Comment (PradumanPannu) [reply howtoplay.html]: we can separate it but there is no need for this because it is just a hyperlink.

Comment (niccokunzmann) [reply howtoplay.html]: You can also use href="index.html" here.

**Comment by niccokunzmann** [comment]
Thanks!

**Pull request merged**